docs/source/history.rst

```diff
@@ -0,0 +1,43 @@
+History
+=======
+
+The history and versioning information is exposed using the @history endpoint.
+It lists the historical versions of the content.
+
+See :ref:`content_get_version` for documentation on reading older versions of a resource.
+
+
+GET Resource history
+--------------------
+
+Listing versions and history of a resource:
+
+..  http:example:: curl httpie python-requests
+    :request: _json/history_get.req
+
+.. literalinclude:: _json/history_get.resp
+   :language: http
+
+This following fields are returned:
+
+-  action: the workflow transition id, or 'Edited' for versioning.
+- actor: the user who performed the action. This contains a subobject with the details.
+- comments: a changenote
+- content_url: link to the content endpoint of this specific version.
+- may_revert: true if the user has permission to revert.
+- time: when this action occured in ISO format.
+- transition_title: the workflow transition's title, for 'Edited' for versioning.
+- type: 'workflow' for workflow changes, 'versioning' for editing, or null for content creation.
+- version_id: identifier for this specific version of the resource.
+
+
+PATCH revert historical versions
+--------------------------------
+
+Reverting to older versions of a resource can be done by issueing a PATCH request with a version_id.
```

Comment:
@tisto yep, I ran into this too.

Maybe if we issue a PATCH on the actual resource with version_id?
In the end it is up to the server to handle a request the way it wants too. So, if we get a PATCH {'version_id': 3} on a resource, the server may -as a side effect- revert to the specified version.

A `POST @history_revert {'version_id': x}` might be an option.